tasks/eth/018-granite-upgrade/README.md

```diff
@@ -0,0 +1,49 @@
+# Mainnet Granite Upgrade + Fault Proof Fixes
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+This is a playbook for upgrading Fault Proofs on Mainnet for Granite compatibility.
+
+This patch also upgrades Fault Proof contracts to fix issues found highlighted by various audits.
+
+The proposal was:
+
+- [x] Posted on the governance forum [here](https://gov.optimism.io/t/upgrade-proposal-10-granite-network-upgrade/8733).
+- [x] Approved by Token House voting [here](https://vote.optimism.io/proposals/46514799174839131952937755475635933411907395382311347042580299316635260952272).
+- [ ] Not vetoed by the Citizens' house [here](https://snapshot.org/#/citizenshouse.eth/proposal/0xb0c109d7f68d3cb1054a50f55556d1820e517129b4b53774cb9ca32e0eabe3a4).
+- [ ] Executed on OP Mainnet.
+
+The governance proposal should be treated as the source of truth and used to verify the correctness of the onchain operations.
+
+Governance post of the upgrade can be found at https://gov.optimism.io/t/upgrade-proposal-10-granite-network-upgrade/8733.
+
+
+## Pre-deployments
+
+- `FaultDisputeGame` - [`0xD9d616E4a03a8e7cC962396C9f8D4e3d306097D3`](https://etherscan.io/address/0xA6f3DFdbf4855a43c529bc42EDE96797252879af).
+- `PermissionedDisputeGame` - [`0x98E3F752c7224F8322Afa935a4CaEC3832bB25c9`](https://etherscan.io/address/0x050ed6F6273c7D836a111E42153BC00D0380b87d).
+- `AnchorStatRegistry` - [`0x666D2f5316B8562e9F7B74D0B72a980E8E6F8D5C`](https://etherscan.io/address/0x1B5CC028A4276597C607907F24E1AC05d3852cFC).
+- `StorageSetter` - [`0x54F8076f4027e21A010b4B3900C86211Dd2C2DEB`](https://etherscan.io/address/0xd81f43eDBCAcb4c29a9bA38a13Ee5d79278270cC).
```

Comment:
The anchor text doesn't match the URLs for these onese.

Reply:
Fixed.